iam_builder/iam_builder.py

```diff
@@ -119,4 +120,11 @@ def build_iam_policy(config: dict) -> dict:  # noqa: C901
             iam_lookup["cloudwatch_athena_query_executions"]
         )
 
+    if "is_cadet_deployer" in config:
```

Comment:
Just checking my understanding of the chunk below:
- is_cadet_deployer API permissions defined in templates.py
- We are checking if 'cadet' is available in the Airflow role name specifically, which is defined in the [role_name].yaml file in the Airflow repo itself
- Only roles with 'cadet' in the name can be cadet deployers.

Correct?

Reply:
Yeah, that's correct